integration-libs/cds/package.json

```diff
@@ -28,6 +28,7 @@
     "@angular/core": "^10.1.0",
     "@angular/router": "^10.1.0",
     "@ngrx/store": "^10.0.0",
+    "@spartacus/checkout": "3.3.0",
```

Comment:
also need to add it to `.release-it.json`.

Reply:
Thanks, good catch.